solana/programs/token-bridge-relayer/src/lib.rs

```diff
@@ -0,0 +1,189 @@
+mod error;
+mod message;
+mod processor;
+mod state;
+
+use anchor_lang::prelude::*;
+use processor::*;
+
+type TargetChainGas = u64;
+type KiloLamports = u64;
```

Comment:
Having only these two, lonely types here seems weird. I think it's probably better, given Anchor's limitations, to just drag around type info in variable names.

Reply:
Done